src/app/charts/chart.component.ts

```diff
@@ -51,7 +51,10 @@ export class ChartComponent implements OnChanges {
             indicator: this.chart.indicator,
             scenario: this.scenario,
             city: this.city,
-            climateModels: this.models
+            climateModels: this.models,
+            // As a temporary solution, the time agg defaults to the 1st valid option.
```

Comment:
I don't think it has be discussed, but now that we do time aggs that seems like a necessary feature to lab.